### PR TITLE
Add comms to README

### PR DESCRIPTION
**Pull request by EzraWeller**
The marketing team is hoping to add some general comms to these READMEs. Let us know if this doesn't work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,11 +3,32 @@
...
 
-Arc is the base layer of the DAO stack. It consists a set of smart contracts deployed on the Ethereum blockchain that define the basic building blocks and standard components that can be used to implement any DAO.
+Arc is the base layer of the DAO stack. It consists of a set of smart contracts deployed on the Ethereum blockchain that define the basic building blocks and standard components that can be used to implement any DAO.
+
+Arc is a modular, upgradeable platform that allows for a rapid natural selection of governance systems.
+
+![Arc Structure Diagram](https://user-images.githubusercontent.com/5414803/51004260-d7252180-1507-11e9-9be7-2018dbc7452d.jpeg)
+*<p align="center">Figure 1: Arc's structure</p>*
+
+**Every box in Figure 1 represents a smart contract** (to be precise, a proxy + logic contract).
```
Comment (orenyodfat) [inline README.md]: we still do not have proxies

Comment (EzraWeller) [reply README.md]: I see -- would it be best to just remove the parenthetical then?

Comment (orenyodfat) [reply README.md]: yes. remove it.

Comment (EzraWeller) [reply README.md]: Done.

**Pull request merged**